rep-2000.rst

```diff
@@ -909,15 +909,15 @@ Dependency Requirements:
 +-------------+----------------+----------------+----------+---------------+------------------+---------------------+
 | OpenCV      |     4.5.4      |     4.5.4      |  3.4.6   |     4.2.0     |      4.5.1       |  4.1.0 / 3.2.0***   |
 +-------------+----------------+----------------+----------+---------------+------------------+---------------------+
-| OpenSSL     |     1.1.1l     |     1.1.1l     |  1.1.1g  |     1.1.1f    |      1.1.1i      | 1.1.1d / 1.1.1b***  |
+| OpenSSL     |     1.1.1l     |     1.1.1l     |  1.1.1k  |     1.1.1f    |      1.1.1i      | 1.1.1d / 1.1.1b***  |
 +-------------+----------------+----------------+----------+---------------+------------------+---------------------+
 | Python      |     3.10.0     |     3.8.3      |  3.6.8   |     3.8.2     |      3.9.1       |  3.8.2 / 3.7.5***   |
```

Comment:
Should the minimum Python version listed above for Humble be lowered to `3.6`?

Reply:
Hmm, that's a good point. @clalancette and @audrow, it looks like the original commit in the original PR listed the minimum Python version as 3.8.

Reply:
Yeah, agreed, we should lower that to 3.6.

Reply:
Fixed in https://github.com/ros-infrastructure/rep/pull/348.